### DISCUSSION TITLE
How is the time from symptoms to hospital admissions handeled?

### DISCUSSION THREAD

**Discussion by Christian-G-UT** [neherlab/covid19_scenarios#310] [General]
I couldn't easily find an answer as to how the time from onset of symptoms to hospital admission is handled in the simulation. Are people directly attributed to hospital at the onset of symptoms or is there a delay assumed? I think this delay could be a tunable (user-defined) parameter of the simulation.

**Comment by nnoll** [comment]
There is a delay currently. Once an individual transitions into the "infectious" state, they will reside there, on average, for the infectious period long (tunable in the epidemiological parameters). The waiting time in this state is distributed with an exponential distribution. So the mean/expected delay from onset of symptoms to hospitalizations *is* tunable, but the distribution is not.

**Reply by Christian-G-UT** [reply]
Thanks for the reply. Let me reformulate to see if I understood it correctly. Once an individual transitions in the "infectious" state he remains there for the infectious period time (which is based on a draw from an exponential distribution with user defined mean). At the end of the infectious period the individual either transitions to recovered or to hospitalized. Is that right?
Thanks again.

**Reply by nnoll** [reply]
Yes, you have it exactly but it's important to note we are simulating the mean of such a stochastic process. Which means if you did such a simulation (that simulated individual people) many times, and took the average of the results, you would obtain our dynamics.

**Reply by Christian-G-UT** [reply]
Thanks for the confirmation. My feeling is that there should be an additional parameter, as I don't think it's realistic to assume individuals being already hospitalized after the infectious period (which on average is only about 3 days). Given the current data it appears reasonable to assume the infectious period is quite short on average (around 3 days), as most people will realize they are thick and isolate themselves quickly to avoid spreading it further. However, based on what I read and hear it does not appear that the severe cases are hospitalized already after 3 days on average after onset of symptoms. In many cases it appears that there is about a week or more between onset of symptoms and time when the disease becomes severe and requires hospitalization. As the number of hospitalized and ICU patients is probably the most reliable terrain data we currently have, trying to fit the model to this data while not properly accounting for the delay between onset of symptoms and time when disease becomes severe leading to hospitalization, will probably lead to inaccurate results. My suggestion would be to introduce a user-defined parameter from time of onset of symptoms to hospitalization (or from time of end of infectiousness to hospitalization).

**Reply by rneher** [reply]
A flexible delay here could indeed be useful. But criteria for hospitalization will depend strongly on country and pressure on the health care system. We hope that critical condition is less ambiguous. Your point is well taken, but we need to strike a balance between too many knobs to turn vs realism. Thanks for the input though, we'll discuss this with our clinical partners.
